packages/exo/CHANGELOG.md

```diff
@@ -3,6 +3,16 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [1.5.0](https://github.com/endojs/endo/compare/@endo/exo@1.4.0...@endo/exo@1.5.0) (2024-05-06)
+
+
+### Features
+
+* **pass-style,exo:** exo boundary only throws throwables ([#2266](https://github.com/endojs/endo/issues/2266)) ([2f0888e](https://github.com/endojs/endo/commit/2f0888e789edca35de86fa9726e6bbd70af8be2f)), closes [#2223](https://github.com/endojs/endo/issues/2223)
```

Comment:
#2223 is a PR, ***NOT*** and issue. #2266 does not state that it closes #2223 . It "refs:" #2223 and says it "is a modest step towards #2223". If this summary was automatically generated from the PR comment, that would be bad. It means that all other such automatically generated summaries are suspect. Do you understand what happened here?

Reply:
This is the same hard-to-fix problem discussed below. Ok for this release to proceed without fixing this. But we need to understand why it is generating the wrong thing, and fix something so it does not next time.

Resolving.